src/AR/objects/Instruments/InstrumentFactory.ts

```diff
@@ -127,7 +127,7 @@ export default class InstrumentFactory {
           object.children[0],
           name,
           keypoints,
-          [10, 10, 10]
+          [13, 13, 13]
```

Comment:
it is too big in my opinion (probably relative to scene size)

Reply:
I revert it. It will be fixed in #54 anyaway.